4_visualize.R

```diff
@@ -0,0 +1,44 @@
+source('4_visualize/src/plot_results.R')
+
+p4 <- list(
+  # Plots for each lake-gcm combo
+  tar_target(
+    p4_20yr_gcm_preds_ice,
+    plot_20yr_gcm_preds_ice(p2_glm_uncalibrated_run_groups, 
+                    outfile_template <- '4_visualize/out/Site_20yr_preds_ice_%s_%s.png'),
+    pattern = map(p2_glm_uncalibrated_run_groups),
+    format='file'),
+  
+  # Plots for each lake
+  tar_target(
+    p4_20yr_average_preds_ice,
+    plot_20yr_average_preds_ice(p2_glm_uncalibrated_lake_groups,
+                                outfile_template <- '4_Visualize/out/Site_20yr_preds_ice_%s.png'),
+    pattern = map(p2_glm_uncalibrated_lake_groups),
+    format = 'file'
+  ),
+  
+  tar_target(
+    p4_20yr_average_profiles,
+    plot_20yr_average_profiles(p2_glm_uncalibrated_lake_groups,
+                               plot_month_days = c('02-15','05-15','08-15','11-15'),
+                               outfile_template <- '4_Visualize/out/Site_20yr_profiles_%s.png'),
+    pattern = map(p2_glm_uncalibrated_lake_groups),
+    format = 'file'
+  ),
+  
+  # Plots for all lakes (diagnostic plots)
+  tar_target(
+    p4_temp_violin_gcms,
+    plot_temp_violin_gcms(p2_glm_uncalibrated_run_groups,
+                            outfile = '4_visualize/out/temperature_violin_gcms.png'),
+    format='file'
+  ),
+  
+  tar_target(
+    p4_temp_violin_months,
+    plot_temp_violin_months(p2_glm_uncalibrated_run_groups,
```

Comment:
This one ([link directly to example](https://user-images.githubusercontent.com/54007288/160445334-b3a94f79-5157-4751-b76a-f7dfbddc3bb7.png) for context) does not seem to vary at all between GCMs. Is this likely because we are just using 5 sites and we think this would be more revealing and useful when we add all of them? Because if not, maybe we just use the previous violin plot.

Reply:
I'm honestly not sure... It might be slightly more variable with more sites, but my guess is that there would not be all that much variation.

Reply:
Going to keep for now, since it just adds a target, and not an additional function.